### ISSUE TITLE
Consider using react-admin for a responsive admin page

### ISSUE THREAD

**Issue by MoralCode** [MoralCode/ClassClock#26]
https://github.com/marmelab/react-admin provides a responsive, react UI that is similar to the twitter settings page layout that I was thinking of using for the Admin panel. This may help make the admin page easier to use and honestly keeping the color changing background for the admin panel is not that important.

Since ClassClock is not really being actively maintained anymore, this is just here as an idea.

Might also need to figure out how to disable the default login page for react-admin and replace it with the Auth0 that ClassClock is using currently

**Comment by MoralCode** [comment]
currently waiting on an answer to this: https://stackoverflow.com/questions/70723255/how-do-i-add-a-base-url-to-all-react-admin-pages

**Comment by MoralCode** [comment]
Progress is going quite well with implementing react-admin. I just pushed a dedicated feature branch for it called `react-admin`.

Remaining TODOs (in no particular order):

- [ ] test and confirm that the server/API is able to receive PATCH queries to uodate data
- [ ] set up UI for creating new schedules
- [ ] confirm that new schedules can be created (POST queries)
- [ ] figure out how the user interface should work for editing the days that each schedule is configured for. (calendars are a react-admin premium feature so this may be hard to do on the cheap)

**Comment by MoralCode** [comment]
Some additional TODOs:

- [ ] figure out why performance and UX makes things feel slow when duplicating schedule items
- [ ] figure out why performance is slow when fetching lots of school schedules

**Comment by MoralCode** [comment]
Admin page been merged to main/ Closing and moving improvements to new issue
